linode_api4/linode_client.py

```diff
@@ -378,32 +378,47 @@ def __setattr__(self, key, value):
 
         super().__setattr__(key, value)
 
-    def image_create(self, disk, label=None, description=None):
+    def image_create(self, disk, label=None, description=None, tags=None):
         """
         .. note:: This method is an alias to maintain backwards compatibility.
                   Please use :meth:`LinodeClient.images.create(...) <.ImageGroup.create>` for all new projects.
         """
-        return self.images.create(disk, label=label, description=description)
+        return self.images.create(
+            disk, label=label, description=description, tags=tags
+        )
 
     def image_create_upload(
-        self, label: str, region: str, description: str = None
+        self,
+        label: str,
+        region: str,
+        description: str = None,
+        tags: List[str] = None,
     ) -> Tuple[Image, str]:
         """
         .. note:: This method is an alias to maintain backwards compatibility.
                   Please use :meth:`LinodeClient.images.create_upload(...) <.ImageGroup.create_upload>`
                   for all new projects.
         """
 
-        return self.images.create_upload(label, region, description=description)
+        return self.images.create_upload(
+            label, region, description=description, tags=tags
+        )
 
     def image_upload(
-        self, label: str, region: str, file: BinaryIO, description: str = None
+        self,
+        label: str,
+        region: str,
+        file: BinaryIO,
+        description: str = None,
```

Comment:
```suggestion
        description: Optional[str] = None,
```

Same rule actually applied to `str` as well, lol

Reply:
I find there are many usage of `somestring: str = None` in our code. We probably need to fix all of them in another PR target dev, because this one will only merge into the feature branch.